Example/TrustSDK/ViewController.swift

```diff
@@ -48,16 +49,34 @@ class ViewController: UIViewController {
             }
         }
     }
-    
+
+    @IBAction func signEthereumMessage(_ sender: UIButton) {
+        let data = Data("Some message".utf8)
+        let message = Data("\u{19}Ethereum Signed Message:\n\(data.count)".utf8)
```

Comment:
can we handle this on app side to avoid adding `CryptoSwift` lib?

Reply:
does iOS crypto supports sha3?

Reply:
This is the test **Example** app, we don't need it in Trust

Reply:
no sha3 in CryptoKit :(